listenbrainz/webserver/templates/user/music_services.html

```diff
@@ -13,8 +13,11 @@ <h3 class="panel-title">Spotify</h3>
             <div class="panel-body">
                 <p>
                     Connect to your spotify account to read your listening history, play music on ListenBrainz (requires
-                    Spotify Premium) or both. We encourage users to choose both options for the best experience but you
+                    Spotify Premium) or both. We encourage users to choose both options for the best experience, but you
                     may also choose only one option:
+                    Make sure your browser is allowing autoplaying media on listenbrainz.org.
+                    If you have a Spotify account connected, you'll have much better results. Otherwise, you'll be limited to 30 second excerpts.
```

Comment:
```suggestion
                    If you have a Spotify account connected, you'll have much better results. Otherwise, we will search for a match on Youtube.
```

This is left over from before, we don't do 30sec previews anymore.